openmetadata-ui/src/main/resources/ui/src/components/SearchDropdown/SearchDropdown.tsx

```diff
@@ -65,14 +67,18 @@ const SearchDropdown: FC<SearchDropdownProps> = ({
   index,
   independent = false,
   hideCounts = false,
+  hasNullOption = false,
 }) => {
   const tabsInfo = searchClassBase.getTabsInfo();
   const { t } = useTranslation();
 
   const [isDropDownOpen, setIsDropDownOpen] = useState<boolean>(false);
   const [searchText, setSearchText] = useState('');
-  const [selectedOptions, setSelectedOptions] =
-    useState<SearchDropdownOption[]>(selectedKeys);
+  const [selectedOptions, setSelectedOptions] = useState<
+    SearchDropdownOption[]
+  >([]);
+  const [nullOptionSelected, setNullOptionSelected] = useState<boolean>(false);
+  const nullLabelText = `No ${label}`;
```

Comment:
Are we not using localization for the label here?

Reply:
We are not doing localisation in Quick Filters title as well, so didn't do here.

Reply:
Fixed